amethyst_gltf/src/format/importer.rs

```diff
@@ -0,0 +1,328 @@
+use std;
```

Comment:
I hope the burden of writing your own loader wasn't oo annoying. It's a compromise between being high-level and not 'doing too much' on behalf of the user.

Reply:
No, it was fine, I started with using the importer to get the first implementation in place, then I  basically copied the importer code and replaced the file loading parts only.